ansible_collections/arista/avd/tests/unit/plugins/plugin_utils/test_ipaddressing.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import absolute_import, division, print_function
```

Comment:
why do we import these?

Reply:
This is to support previous python versions as it is followed in other modules.

Reply:
`print_function` as far as I know is to support `print(..)` format in python 2 rather than `print ...` so I am not sure we would really need it.

Let's maybe try to understand what these do nd see if we can get rid of them?

Reply:
@gmuloc this is ansible boilerplate enforced by ansible-test for _everything_ that could potentially run on a target host. Depending on where we import this (in modules or filters) it will be enforced. @Vibhu-gslab you can try to remove it and see if it fails `ansible-test sanity`.

Reply:
ah you did :)

Reply:
same for metaclass

Reply:
hmmm fair enough .. let's leave them in and wait for ansible to catch up on python2 deprecation.. apologies for the noise @Vibhu-gslab

Reply:
@gmuloc no worries :), I will still try to remove it and see if sanity passes as suggested by @ClausHolbechArista .
Can you please approve running workflows in the PR. Thanks.

Reply:
what was the conclusion here?

Reply:
I am getting warning on running ansible-test sanity (WARNING: Skipping sanity test "compile" on Python 3.8 because it could not be found.) I have pushed the code after removing future and metaclass, to know for sure if we can run all CI checks it would be best (need approval for the same).

Reply:
All the tests have passed in CI after removing "future and metaclass" :)

Reply:
perfect. Then we can clean up the other test files later. Thanks